_DeploymentAndDistroPackaging/ansible/roles/ciao-launcher/tasks/install.yml

```diff
@@ -37,5 +37,14 @@
       - "{{ agent_key }}"
     notify: restart launcher
 
+  - name: Get OVMF firmware
+    become: no
+    connection: local
+    get_url:
+      url: "{{ firmware_image_url }}"
+      dest: "{{ workdir }}/OVMF.fd"
+      validate_certs: no
+      force: no
+
   - name: Copy OVMF firmware
     copy: src={{ workdir }}/OVMF.fd dest=/usr/share/qemu/OVMF.fd
```

Comment:
Not related to this patch, but related to my comments in the other patch in this series:  what is the mode of the created /usr/share/qemu/OVMF.fd file?  I prefer an explicit 0644 over something implicit.